crates/fiber-lib/src/fiber/network.rs

```diff
@@ -860,7 +868,7 @@ pub enum NetworkActorEvent {
     FundingTransactionFailed(OutPoint),
 
     /// A closing transaction has been confirmed.
-    ClosingTransactionConfirmed(PeerId, Hash256, Byte32, bool),
+    ClosingTransactionConfirmed(PeerId, Hash256, Byte32, bool, bool),
```

Comment:
please add commands for fields, otherwise it's hard to understand the meaning of them.